source/requirements.rst

```diff
@@ -0,0 +1,111 @@
+* For the present, nothing is ever deleted. In the future, we will add the
+  capability for beamline scientists to selectively delete data or for the
+  facility to "garbage collect" old data that has not speicfically been marked
+  for long-term archival. However, this will not be implemented until all runs
+  are clearly identified with an owner, which relies on guidance from the user
+  task force, as outlined previously.
+* Suitcase will provide tools for exporting from the databroker to various file
+  formats (SPEC files and HDF5 are currently support). We are not advocating
+  these formats as standards in any way: they are merely conveniences for users
+  who rely on a file-based workflow and are not prepared to write their own
+  custom export code. In general they are lossy and do not capture the full
+  information or structure of the original documents.
+
+Python Software Dependencies
+============================
+
+* Data collection software will require Python 3. Presently it requires
+  3.4 or greater, but may be updated to rely on later minor releases
+  (e.g. Python 3.6). Minor upgrades are expected to require little or no
+  change to user code.
+* Data analysis software will support both Python 2.7 and Python 3.4+ for now.
+  Following the lead of the
+  `scientific Python community at large <http://www.python3statement.org/>`_,
+  support for 2.7 will be dropped before 2020, when Python 2.7 itself will stop
+  receiving security patches.
+* Users often interact with the software using IPython, but no core
+  functionality should rely on IPython. Though IPython integration such as
+  "magics" may be created for convenience and packaged with the software, all
+  important functions should be runnable from a plain Python script and should
+  not assume IPython is installed.
+* In general, if an acceptable implementation of some functionality exists in a
+  widely-used and actively-supported library, we prefer reusing that
+  implementation over rewriting. In evaluating an implementation, we look for
+  separation of I/O and visualization code from the core scientific logic, good
+  community practices like open development and automated unit testing,
+  reasonable packaging and dependencies, and a design that makes individual
+  components easy to pick apart.
+
+Python Software APIs
+====================
+
+* Ophyd will provide a low-level API for debugging hardware and whatever
+  high-level API is needed by bluesky.
+* Bluesky will interact with hardware only through ophyd's high-level API. It
+  will provide general-purpose "plans" and the tools to assemble more
+  beamline- and experiment-specific plans.
+* Databroker will provide a user-friendly programmatic interface for retrieval
+  documents and deferencing data from files indexed by filestore.
+* The documents emmited by bluesky and the documents returned by databroker
+  should be compatible (i.e., equal).
```

Comment:
`emmited`, and would something like "the document keys emitted by bluesky are guaranteed to be a subset (possibly a strict subset) of those returned by the data broker" be more accurate?